api/loader/getter.go

```diff
@@ -58,32 +58,41 @@ func newLoaderAtGetter(raw string, fSys filesys.FileSystem, referrer *fileLoader
 func getRemoteTarget(rs *remoteTargetSpec) error {
 	var err error
 
-	rs.TempDir, err = filesys.NewTmpConfirmedDir()
+	// Get the pwd
+	pwd, err := os.Getwd()
 	if err != nil {
-		return err
+		log.Fatalf("Error getting wd: %s", err)
 	}
 
-	rs.Dir = filesys.ConfirmedDir(rs.TempDir.Join("repo"))
+	// Normalize the url
+	source, err := getter.Detect(rs.Raw, pwd, []getter.Detector{
+		new(getter.GitHubDetector),
+		new(getter.GitDetector),
+		new(getter.BitBucketDetector),
```

Comment:
We actually use a [fork of go-getter](https://github.com/yujunz/go-getter) which hasn't been updated to have the gitlab detector yet.

Reply:
The forks has been updated with gitlab detector.

Reply:
I've updated go-getter and added GitLabDetector, note that this only supports top-level urls. So only gitlab.com/foo/bar.git//manifests but a url of gitlab.com/foo/bar/baz.git//manifests will fail. Also note that since the git cloner seems like it does not support unqualified gitlab urls, that's another point of difference. That said, it doesn't support bitbucket either so I guess there already is some difference here anyway.